docs/installconfig/configure.rst

```diff
@@ -4,12 +4,21 @@ Configuring ToDD
 ToDD uses configuration files (typically found in ``/etc/todd``) to control it's behavior. The server and the agent use their own individual config files, and sample files are shown below.
 
 .. NOTE:: 
+
    Configuration files for both ``todd-server`` and ``todd-agent`` binaries load once at initial start. So, in order to apply any changes to the configuration, these processes need to be restarted.
 
 Server Configuration
 --------------------
 
-The server configuration file (usually ``/etc/todd/server.cfg``) contains configurations for all services that the server will require - things like integrated databases, communications plugins, and internal calculations. Here is a sample server configuration file, with comments inline:
+The server configuration file (usually ``/etc/todd/server.cfg``) contains configurations for all services that the server will require - things like integrated databases, communications plugins, and internal calculations.
+
+.. WARNING:: 
+
+   Pay particular note to he LocalResources section of the configuration. The DefaultInterface option is required (or in lieu thereof, the IPAddrOverride option) so that the server will know what IP address to serve assets from to the agents. The server will not successfully start if an address is not determined.
```

Comment:
I would write "The server will not successfully start if an address is not determined" as
"The server will not start if it cannot determine its IP address." Minor style thing though, so it's not important.